uniform_webview/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I see many different package.json files in this PR, what's their purpose?

Reply:
They're for different js builds. Theres one each for:
* The main extension build
* The "server" build used for interfacing with slang
* The main webview that renders the playground run
* The uniform components webview
* The files that are shared across multiple builds
* The extension itself to give vscode information

Since they have different dependencies/environments they get different package.json files.